packages/jest-types/src/Global.ts

```diff
@@ -123,5 +123,5 @@ export interface GlobalAdditions extends TestFrameworkGlobals {
 export interface Global
   extends GlobalAdditions,
     Omit<typeof globalThis, keyof GlobalAdditions> {
-  [extras: string]: unknown;
+  [extras: string | symbol]: unknown;
```

Comment:
Ah.. Got it. That’s good idea, it will also allow user to have `this.global[someSymbol]` inside environment. Perhaps this is good place for TS built-in `PropertyKey` type (which is `type PropertyKey = string | number | symbol`):

```suggestion
  [extras: PropertyKey]: unknown;
```